OTAcceleratorCore/OTMultiPartyCommunicator.h

```diff
@@ -169,4 +169,19 @@ typedef void (^OTMultiPartyCommunicatorBlock)(OTCommunicationSignal signal, OTMu
  */
 @property (readonly, nonatomic) OTStreamVideoType videoType;
 
+/**
+ *  An integer value to indicate the total connections.
+ */
+@property (readonly, nonatomic) NSUInteger connectionCount;
+
+/**
+ *  A boolean value to indicate if the connection is the oldest.
+ */
+@property (readonly, nonatomic) BOOL isFirstConnection;
+
+/**
+ *  A string value to indicate the own connectionId.
+ */
+@property (readonly, nonatomic) NSString *selfConnectionId;
```

Comment:
IMHO, I don't suggest to introduce this as you can easily get this info from `OTAccelerator.connection.connectionId`.